test/helpers/wrappers.go

```diff
@@ -49,6 +49,14 @@ func CurlFail(endpoint string) string {
 		CurlConnectTimeout, endpoint)
 }
 
+// CurlWithHTTPCode retunrs the string representation of the curl command which
+// only outputs the HTTP code returned by its execution against the specified
+// endpoint.
+func CurlWithHTTPCode(endpoint string) string {
+	return fmt.Sprintf(`curl -s --output /dev/stderr -w '%%{http_code}' --connect-timeout %d XGET %s`,
+		CurlConnectTimeout, endpoint)
+}
```

Comment:
This isn't used anymore since I removed the Ginkgo changes I made, but still could prove useful, so I decided to keep it around.